<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ytdl</class>
 <widget class="QMainWindow" name="ytdl">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ytdl</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>600</width>
      <height>121</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Music">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Music&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Music&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <attribute name="title">
      <string>Audio</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="labelQuality">
          <property name="maximumSize">
           <size>
            <width>105</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Quality:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="radiosQuality">
          <item>
           <widget class="QRadioButton" name="radioBest">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Best</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">MQualityGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioGood">
            <property name="text">
             <string>Good</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">MQualityGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioMedium">
            <property name="text">
             <string>Medium</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">MQualityGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioLow">
            <property name="text">
             <string>Low</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">MQualityGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="labelMFormat">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Format:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="radiosMFormat">
          <item>
           <widget class="QRadioButton" name="radioAAC">
            <property name="text">
             <string>AAC</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">MFormatGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioOGG">
            <property name="text">
             <string>OGG</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">MFormatGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioMP3">
            <property name="text">
             <string>MP3</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">MFormatGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioOPUS">
            <property name="text">
             <string>OPUS</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">MFormatGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioWAV">
            <property name="text">
             <string>WAV</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">MFormatGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="Video">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Video</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="labelResolution">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Max Resolution:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="radiosResolution">
          <item>
           <widget class="QRadioButton" name="radio4K">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>78</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>4K</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">VResGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio1440p">
            <property name="text">
             <string>1440p</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">VResGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio1080p">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>87</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>87</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>1080p</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">VResGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio720p">
            <property name="text">
             <string>720p</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">VResGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radio480p">
            <property name="text">
             <string>480p</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">VResGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="labelVFormat">
          <property name="maximumSize">
           <size>
            <width>135</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Preferred Format:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="radiosVFormat">
          <item>
           <widget class="QRadioButton" name="radio3GP">
            <property name="text">
             <string>3GP</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">VFormatGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioFLV">
            <property name="text">
             <string>FLV</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">VFormatGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioMP4">
            <property name="text">
             <string>MP4</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">VFormatGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioWEBM">
            <property name="text">
             <string>WEBM</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">VFormatGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="urlLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelURL">
       <property name="text">
        <string>Enter URL:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineURL"/>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="playlistCheck">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Playlist</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>581</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="downloadLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonBrowse">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineBrowse">
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="defaultsCheck">
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDownload">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="MQualityGroup"/>
  <buttongroup name="VFormatGroup"/>
  <buttongroup name="VResGroup"/>
  <buttongroup name="MFormatGroup"/>
 </buttongroups>
</ui>
